src/app/components/ScrollablePromo/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,438 @@
+    .emotion-0 {
+      grid-template-columns: repeat(10, 1fr);
+      grid-column-end: span 10;
+      grid-column-start: 5;
+    }
+  }
+}
+
+.emotion-2 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  overflow-wrap: break-word;
+  list-style: none;
+  padding-right: 0;
+  margin: 0;
+  overflow-x: scroll;
+  scroll-behavior: auto;
+  -webkit-overflow-scrolling: touch;
+  scrollbar-width: none;
+  -ms-overflow-style: none;
+}
+
+.emotion-2::-webkit-scrollbar {
+  display: none;
+}
+
+.emotion-4 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-shrink: 0;
+  -ms-flex-negative: 0;
+  flex-shrink: 0;
+}
+
+@media (min-width: 0rem) {
+  .emotion-4 {
+    margin-right: 0.5rem;
+  }
+
+  .emotion-4:first-child {
+    margin-right: 0.5rem;
+  }
+
+  .emotion-4:last-child {
+    margin-left: 0.5rem;
+  }
+}
+
+@media (min-width: 25rem) {
+  .emotion-4 {
+    margin-right: 1rem;
+  }
+
+  .emotion-4:first-child {
+    margin-right: 1rem;
+  }
+}
+
+@media (min-width: 63rem) {
+  .emotion-4 {
+    margin-right: 1rem;
+  }
+
+  .emotion-4:first-child {
+    margin-right: 0;
+  }
+}
+
+.emotion-6 {
+  margin-bottom: 1.5rem;
+  background-color: #FFFFFF;
+  padding: 1rem;
+}
+
+@media (min-width: 0rem) {
+  .emotion-6 {
+    width: 14.8125rem;
+  }
+}
+
+@media (min-width: 37.5rem) {
+  .emotion-6 {
+    width: 11.125rem;
+  }
+}
+
+@media (min-width: 63rem) {
+  .emotion-6 {
+    width: 12.6875rem;
+  }
+}
+
+.emotion-8 {
+  width: 100%;
+  text-overflow: ellipsis;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  color: #202224;
+}
+
+.emotion-8:hover,
+.emotion-8:focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.emotion-8:visited {
+  color: #545658;
+}
+
+<div>
+  <div
+    class="emotion-0 emotion-1"
+    dir="ltr"
+  >
+    <ul
+      class="emotion-2 emotion-3"
+      role="list"
+    >
+      <li
+        class="emotion-4 emotion-5"
+      >
+        <div
+          class="emotion-6 emotion-7"
+        >
+          <a
+            class="emotion-8 emotion-9"
+            href="https://www.bbc.co.uk/pidgin"
+          >
+            Show headline
+          </a>
+        </div>
+      </li>
+      <li
+        class="emotion-4 emotion-5"
+      >
+        <div
+          class="emotion-6 emotion-7"
+        >
+          <a
+            class="emotion-8 emotion-9"
+            href="https://www.bbc.com/mundo"
+          >
+            Link with Headline
+          </a>
+        </div>
+      </li>
+      <li
+        class="emotion-4 emotion-5"
+      >
+        <div
+          class="emotion-6 emotion-7"
+        >
+          <a
+            class="emotion-8 emotion-9"
+            href="https://www.bbc.com/brasil"
+          >
+            https://www.bbc.com/brasil
```

Comment:
A url? What content will we have in storybook?

Reply:
We will display the URL... I know is not good, but I think this is an extreme test case that we created. Where the data given to us has the url as title.

This is how it appears in storybook:
<img width="699" alt="Screenshot 2021-12-16 at 16 22 18" src="https://user-images.githubusercontent.com/90621252/146409504-135d07f0-80e6-45b9-b030-432fe62cd7fc.png">
.